### DISCUSSION TITLE
Copy/Past boxes section

### DISCUSSION THREAD

**Discussion by Alldar** [OFFLINE-GmbH/oc-boxes-support#57] [Ideas and feature requests]
![copy-all](https://user-images.githubusercontent.com/905204/226543897-0d5f50cd-d9da-46bb-97f7-1c5364d6532d.png)

It would be nice to be able to copy a section or all sections of boxes to the clipboard, in order to then paste them to another page. As an option download in json format and then upload with replacement to another page.

This is very useful when you have a multilingual site and pages differ only in translation.

I know that there is a command "boxes:mirror-site", but for the site editors can not use it, they are not programmers.

Thank you for your wonderful product :)

**Comment by tobias-kuendig** [comment]
Thank you for this feedback @Alldar! I can see a few situations were this could be useful. Somethink like a "copy json" and "create from json" feature, that allows you to quickly copy a box from A to B. 

I'll keep this in the backlog and let you know if we plan to implement it.
